packages/plugin-meye-config/src/index.ts

```diff
@@ -0,0 +1,965 @@
+import * as tf from "@tensorflow/tfjs";
+import { JsPsych, JsPsychPlugin, ParameterType, TrialType } from "jspsych";
+
+const info = <const>{
+  name: "meye-config",
+  parameters: {},
+};
+
+type Info = typeof info;
+
+/**
+ * **plugin-meye-config**
+ *
+ * Sets up extension-meye to capture participant pupil diameter.
+ *
+ * @author Adam Vasarhelyi
+ * @see {@link https://../docs/extension-meye}
+ */
+class MeyeConfigPlugin implements JsPsychPlugin<Info> {
+  static info = info;
+
+  constructor(private jsPsych: JsPsych) {}
+
+  trial(display_element: HTMLElement, trial: TrialType<Info>) {
+    console.log("Loaded TensorFlow.js - version: " + tf.version.tfjs); // debug
+
+    // Style sheet
+    const styleSheet = document.createElement("link");
+    styleSheet.setAttribute("rel", "stylesheet");
+    styleSheet.setAttribute("type", "text/css");
+    styleSheet.setAttribute("href", "../../plugin-meye-config/css/main.css");
+    styleSheet.setAttribute("id", "plugin-meye-config-stylesheet");
+    document.head.appendChild(styleSheet);
+
+    // Setup event stuff
+    const passedPerformanceCheck = new Event("passedPerformanceCheck", { bubbles: true });
+    const passedSetup2 = new Event("passedSetup2", { bubbles: true });
+    const exit = new Event("exit", { bubbles: true });
+    const recalibrate = new Event("recalibrate", { bubbles: true });
+
+    display_element.addEventListener("recalibrate", introduction);
+    display_element.addEventListener("dragover", dragover);
+    display_element.addEventListener("dragover", resizeover);
+    display_element.addEventListener("drop", drop); // in common
+
+    display_element.addEventListener("exit", () => {
+      cleanHTML();
+      this.jsPsych.endExperiment();
+    });
+
+    display_element.addEventListener("passedPerformanceCheck", () => {
+      continueBtn.addEventListener("click", () => {
+        display_element.dispatchEvent(passedSetup2);
+      });
+    });
+
+    display_element.addEventListener("passedSetup2", () => {
+      var data = { settings: idealObject };
+
+      cleanHTML();
+      this.jsPsych.finishTrial(data);
+    });
+
+    // Begin plugin
+    styleSheet.onload = () => {
+      introduction();
+    };
+
+    function introduction() {
```

Comment:
i'm wondering if it's just worth leaving this out and instead having a user just use an `html-button-response` or `instructions` plugin beforehand: it would also give them more power to customize this prompt

Reply:
I really wanted it to be part of the plugin because they're special instructions that are unique to the plugin and are unlikely to change or be different between research. The reason it's unlikely to change is because orthogonal lighting is better for it and outdoor lighting is orthogonal. I can still change this if you prefer, though.

Reply:
as i mentioned above with parameters, probably the place that language would come in would be if we were to serve this experiment to someone that doesn't speak english. maybe the special instructions are unlikely to change between different experiments, but it'll make it harder to serve the experiment to those that speak other languages

Reply:
I see, all good. Would you prefer that I remove that part of the instructions or are you suggesting that I add a parameter for localisation? The former option wouldn't fix that in the actual calibration part of the plugin, there are instructions in English.